dts/arm/st/h7rs/stm32h7rs.dtsi

```diff
@@ -492,6 +492,28 @@
 			status = "disabled";
 		};
 
+		xspi1: spi@52005000 {
+			compatible = "st,stm32-xspi";
+			reg = <0x52005000 0x1000>, <0x90000000 0x10000000>;
+			interrupts = <105 0>;
+			clock-names = "xspix";
+			clocks = <&rcc STM32_CLOCK(AHB5, 5U)>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "disabled";
+		};
+
+		xspi2: spi@5200a000 {
+			compatible = "st,stm32-xspi";
+			reg = <0x5200a000 0x1000>, <0x70000000 0x10000000>;
```

Comment:
could be written with with DT_SIZE_K/M()
like requested her  https://github.com/zephyrproject-rtos/zephyr/pull/88646#discussion_r2044450229